### ISSUE TITLE
Not all the code uses ffmpeg.bin

### ISSUE THREAD

**Issue by maztan** [damianociarla/node-ffmpeg#74]
Some code ignores ffmpeg.bin and tries to use ffmpeg from PATH. For example the fnExtractSoundToMP3 function.

**Comment by ChennyBaBy** [comment]
Any luck on solving this? I also tired the .bin method but doesn't seem to pick it up. Also no readme on the .bin

**Comment by maztan** [comment]
@ChennyBaBy, no, you would have to file a pull request or wait for someone else to fix it.

**Comment by damianociarla** [comment]
Fixed in 1.0.0. Every operation, including MP3 extraction, frame extraction, probing, watermarking, and save, now uses the configured ffmpegPath/ffprobePath. No operation has a hard-coded ffmpeg executable.
